sources/packages/backend/apps/api/src/route-controllers/report/_tests_/e2e/report.institutions.controller.exportReport.e2e-spec.ts

```diff
@@ -213,12 +213,31 @@ describe("ReportInstitutionsController(e2e)-exportReport", () => {
               "Has Study Breaks":
                 (!secondSavedOffering.lacksStudyBreaks).toString(),
               "Actual Tuition":
-                secondSavedOffering.actualTuitionCosts.toString(),
+                secondSavedOffering.actualTuitionCosts.toLocaleString("en-US", {
```

Comment:
Can it be simplified as `secondSavedOffering.actualTuitionCosts.toFixed(2)`?

Reply:
updated